src/main/target/DALRCF405/target.c

```diff
@@ -43,5 +43,14 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM4, CH1, PB6,  TIM_USE_LED, 0, 0),     // LED STRIP(1,0)
 
     DEF_TIM(TIM2, CH1, PA5,  TIM_USE_PWM, 0, 0),     // FC CAM	
+// Backdoor timers on UARTs
+    DEF_TIM(TIM9,  CH1,  PA2,  TIM_USE_NONE,                0, 0), // UART2_TX
```

Comment:
Looks good, but can you please open a separate pull request for the updates to the DALRCF405?

Reply:
Ah...I don't know why they merge together :D...
I'll have a try

Reply:
I revert b249d48 ,just keep F7 first.